scrapers.js

```diff
@@ -40,6 +40,28 @@ const scrapers = [
       return counties;
     }
   },
+  {
+    state: 'IL',
+    country: 'USA',
+    priority: 1,
+    url: 'http://www.dph.illinois.gov/sites/default/files/COVID19/COVID19CountyResults.json',
+    async scraper() {
+      const data = await fetch.json(this.url);
+
+      const counties = [];
+
+      for (const county of data.characteristics_by_county.values) {
+        counties.push({
+          county: transform.addCounty(county.County),
+          cases: county.confirmed_cases,
```

Comment:
```suggestion
          cases: parse.number(county.confirmed_cases),
```

Reply:
Chances are it's already a number, but... If they screw it up, might as well not have it die :)